File: 01.01.判定字符是否唯一/01.01.cpp
class Solution {
public:
    bool isUnique(string astr) {
        std::unordered_set<char> uset;
        for (char c : astr) {
            if (uset.count(c) > 0) {
                return false;
            }
            uset.insert(c);
        }
        return true;
    }
};

File: 01.01.判定字符是否唯一/01.01.rs
use std::collections::HashSet;
impl Solution {
    pub fn is_unique(astr: String) -> bool {
        let mut hs: HashSet<char> = HashSet::new();
        for c in astr.chars() {
            if hs.contains(&c) == true {
                return false;
            }
            hs.insert(c);
        }
        return true;
    }
}
